auth-api/src/auth_api/services/authorization.py

```diff
@@ -109,18 +109,27 @@ def get_user_authorizations_for_entity(business_identifier: str, expanded: bool
                     permissions = PermissionsService.get_permissions_for_membership(auth.status_code, 'SYSTEM')
                     auth_response['roles'] = permissions
         else:
-            keycloak_guid = user_from_context.sub
-            if business_identifier and keycloak_guid:
-                auth = AuthorizationView.find_user_authorization_by_business_number(
-                    business_identifier=business_identifier,
-                    keycloak_guid=keycloak_guid,
-                    org_id=user_from_context.account_id
-                )
+            if business_identifier and (org_id := user_from_context.account_id):
```

Comment:
is it possible `org_id` could have been None previously?

Reply:
yeah and if they are a system user it should go through the other 'if' so this should be good